src/common/CMakeLists.txt

```diff
@@ -1,15 +1,14 @@
 add_subdirectory(binaries_op)
-add_subdirectory(bzip2_op)
 add_subdirectory(config)
 add_subdirectory(data_provider)
 add_subdirectory(dbsync)
 add_subdirectory(error_messages)
-add_subdirectory(file_op)
```

Comment:
Please, if not used remove the code of this library. We will replace it in the future if any not migrated component is still using it.

Reply:
It was being used. I've just rebased and that line is no longer present (i didn't remove it).